specification/powerplatform/resource-manager/Microsoft.PowerPlatform/common/v1/definitions.json

```diff
@@ -74,6 +74,10 @@
           "readOnly": true,
           "type": "string",
           "description": "The type of the resource. E.g. \"Microsoft.Compute/virtualMachines\" or \"Microsoft.Storage/storageAccounts\""
+        },
```

Comment:
All the models in this `definitions.json` except the parameters could be replaced by referencing the types.json from common folder. Can you make the change?

Reply:
The Resource model in common types.json is missing systemData which causes LintDiff check to fail. Will wait to hear from you.

Reply:
This is good [example](https://github.com/Azure/azure-rest-api-specs/blob/bb887a1ae1aedbe1e7d044581b3668a8099e30ee/specification/adp/resource-manager/Microsoft.AutonomousDevelopmentPlatform/preview/2021-02-01-preview/adp.json#L731) for reference.